rest/route/commit_queue.go

```diff
@@ -47,3 +47,47 @@ func (cq commitQueueGetHandler) Run(ctx context.Context) gimlet.Responder {
 
 	return gimlet.NewJSONResponse(commitQueue)
 }
+
+type commitQueueDeleteItemHandler struct {
+	project string
+	item    string
+
+	sc data.Connector
+}
+
+func makeDeleteCommitQueueItems(sc data.Connector) gimlet.RouteHandler {
+	return commitQueueDeleteItemHandler{
+		sc: sc,
+	}
+}
+
+func (cq commitQueueDeleteItemHandler) Factory() gimlet.RouteHandler {
+	return &commitQueueDeleteItemHandler{
+		sc: cq.sc,
+	}
+}
+
+func (cq commitQueueDeleteItemHandler) Parse(ctx context.Context, r *http.Request) error {
+	cq.project = gimlet.GetVars(r)["project_id"]
+	cq.item = gimlet.GetVars(r)["item"]
```

Comment:
I'd call gimlet.GetVars(r) once and then just access the value twice...

Reply:
done.